go.mod

```diff
@@ -3,22 +3,24 @@ module github.com/guoger/stupid
 go 1.12
 
 require (
-	github.com/Knetic/govaluate v3.0.0+incompatible // indirect
 	github.com/Shopify/sarama v1.23.1 // indirect
 	github.com/fsouza/go-dockerclient v1.4.4 // indirect
 	github.com/gogo/protobuf v1.2.1
 	github.com/golang/protobuf v1.3.2
 	github.com/hashicorp/go-version v1.2.0 // indirect
-	github.com/hyperledger/fabric v1.4.3
-	github.com/hyperledger/fabric-amcl v0.0.0-20181230093703-5ccba6eab8d6 // indirect
+	github.com/hyperledger/fabric v2.0.1+incompatible
```

Comment:
fabric internalized `comm` pkg in later releases. I think we could actually remove fabric from dependencies entirely (copy some code over)